src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,24 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (object == null || object.getClass() != this.getClass()) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) object;
+        return this.getWidth().equals(rectangle.getWidth())
+                && this.getLength().equals(rectangle.getLength())
+                && this.getColor().equals(rectangle.getColor());
```

Comment:
there is no need to use getters while you are in a class which has this field

Reply:
Ok, fixed.